plugins/kubernetes-backend/README.md

```diff
@@ -8,21 +8,33 @@ It responds to Kubernetes requests from the frontend.
 
 ## Configuration
 
-### clusterLocatorMethod
+### serviceLocatorMethod
 
 This configures how to determine which clusters a component is running in.
 
-Currently, the only valid locator method is:
+Currently, the only valid serviceLocatorMethod is:
 
-#### configMultiTenant
+#### multiTenant
 
 This configuration assumes that all components run on all the provided clusters.
 
+### clusterLocatorMethods
+
+This is used to determine where to retrieve cluster configuration from.
+
+Currently, the only valid serviceLocatorMethod is:
+
+#### config
+
+This clusterLocatorMethod will read cluster information in from config
+
 Example:
 
 ```yaml
 kubernetes:
-  clusterLocatorMethod: 'configMultiTenant'
+  serviceLocatorMethod: 'multiTenant'
+  clusterLocatorMethods:
```

Comment:
What do you think about merging this into the config below instead? Making it more of a `clusterSources`-type thing, each object having its own `type` or `method`. Something like

```yaml
clusters:
  - type: config
    options:
      url: http://127.0.0.1:9999
      name: minikube
      serviceAccountToken: <TOKEN FROM STEP 4>
      authProvider: 'serviceAccount'
  - type: config
    options:
      url: http://127.0.0.2:9999
      name: gke-cluster-1
      authProvider: 'google'
  - type: kubeconfig
    options:
      file: ~/.kube/config
  - type: http # if we want to attempt to standardize this
    options:
      url: http://k8s-discovery-service.internal.example.com/v1/cluster-discovery
```

Reply:
Yes, I think we probably should go in this direction, but I'm going to wait until I try to add the gcloud implementation before coming up with the config values